portal-impl/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: ${manifest.bundle.name}
 Bundle-SymbolicName: ${manifest.bundle.symbolic.name}
-Bundle-Version: 82.0.3
+Bundle-Version: 82.1.0
```

Comment:
Are you sure your change is causing the version increment here?

Reply:
Removed it! Thanks